example/ck_tile/01_fmha/fmha_fwd.cpp

```diff
@@ -276,6 +319,14 @@ bool run(const ck_tile::ArgParser& arg_parser)
         ck_tile::FillTrigValue<VDataType>{}(v_host);
         ck_tile::FillTrigValue<BiasDataType>{}(bias_host);
     }
+    else if(init_method == 3) // suitable for fp8 quantization
```

Comment:
please update `arg_parser` description for the `"init"` option as well

Reply:
c9b59e398b76528a73a784ae91adc19b5fd9d72d